m2m-transfers/V3/REPWRITERSPECS/BNO.INTERACTIVE.PRO

```diff
@@ -250,12 +258,16 @@ PROCEDURE NTADDCROSSACCOUNTS
      NTPREFACCESSID=PREFERENCE ACCESS:ID
      FOR ACCOUNT PREFERENCE ACCESS:ACCOUNTNUMBER
       DO
-       IF NTPREFACCESSIDTYPE=2 OR NTPREFACCESSIDTYPE=0 THEN
+       BNODEBUGMSG=FORMAT("** NTSLTYPE:99:",NTSLTYPE)
+       CALL BNODEBUGMSGADD
+       IF (NTPREFACCESSIDTYPE=2 OR NTPREFACCESSIDTYPE=0) AND
+          (NTCROSSACCTSLTYPE=NTSLTYPESHARE OR
+           NTCROSSACCTSLTYPE=NTSLTYPESHARELOAN) THEN
```

Comment:
Do we also need to have a check for NTSLTYPESHARELOAN up above starting at line #226?

Reply:
@rrobison44 is this what you're referring to - the reference to unrestricted ID Types (line 231)?

![image](https://github.com/Banno/banno-powerons/assets/60237061/d9ab3f7c-b8bf-49dc-a61c-effa2468de4d)